backend/wayland/output.c

```diff
@@ -224,7 +224,6 @@ struct wlr_output *wlr_wl_output_create(struct wlr_backend *_backend) {
 
 	wlr_output->width = 640;
 	wlr_output->height = 480;
-	wlr_output->scale = 1;
```

Comment:
Yeah, it's just a follow-up on that earlier fix from yesterday. We no longer need it here.